src/ElectricPokemon.java

```diff
@@ -0,0 +1,123 @@
+import java.util.List;
+import java.util.Arrays;
+
+public class ElectricPokemon extends Pokemon{
+    private static final String type = "electric";
+    private static final List<String> attacks = Arrays.asList("thunderPunch", "thunder", "electroBall", "VoltTackle");
+
+
+
+    public ElectricPokemon (String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound, type);
+    }
+
+    void thunderPunch(Pokemon name, Pokemon enemy){
+        System.out.println(name.getName() + " attacks " + enemy.getName() + "with a thunderpunch." );
+
+        switch (enemy.getType()) {
+            case "grass":
+                System.out.println(enemy.getName() + "loses 10 hp");
+                enemy.setHp(enemy.getHp() - 10);
+                break;
+            case "fire":
+                System.out.println(enemy.getName() + "loses 5 hp");
+                enemy.setHp(enemy.getHp() - 5);
+                break;
+            case "electric":
+                System.out.println(enemy.getName() + "loses 2 hp");
+                enemy.setHp(enemy.getHp() - 2);
+                break;
+            case "water":
+                System.out.println(enemy.getName() + "loses 15 hp");
+                enemy.setHp(enemy.getHp() - 15);
+                break;
+            default:
+                System.out.println("something went wrong");
+        }
+        System.out.println(enemy.getName() + " has " + enemy.getHp() + " hp left.");
+    }
+
+
+    void thunder(Pokemon name, Pokemon enemy){
+        System.out.println(name.getName() + " attacks " + enemy.getName() + "with a thunder." );
+
+        switch (enemy.getType()) {
+            case "grass":
+                System.out.println(enemy.getName() + "loses 15 hp");
+                enemy.setHp(enemy.getHp() - 15);
+                break;
+            case "fire":
+                System.out.println(enemy.getName() + "loses 10 hp");
+                enemy.setHp(enemy.getHp() - 10);
+                break;
+            case "electric":
+                System.out.println(enemy.getName() + "loses 5 hp");
+                enemy.setHp(enemy.getHp() - 5);
+                break;
+            case "water":
+                System.out.println(enemy.getName() + "loses 25 hp");
+                enemy.setHp(enemy.getHp() - 25);
+                break;
+            default:
```

Comment:
Een default is hier inderdaad goed op zijn plaats. Realistisch zou je hier nooit moeten kunnen komen, maar technisch gezien wel. Je swticht immers op een String en niet op een enum, dus kan het technisch gezien vanalles zijn.